release/Magarena/scripts/Commune_with_Dinosaurs.groovy

```diff
@@ -1,12 +1,13 @@
 def action = {
     final MagicGame game, final MagicEvent event ->
-    final MagicCardList copyTopCards = new MagicCardList(event.getRefCardList());
+    final MagicCardList topCards = new MagicCardList(event.getRefCardList());
     event.processChosenCards(game, {
         final MagicCard chosen ->
+        game.doAction(new RevealAction(chosen));
```

Comment:
Remove the RevealAction and add a LookAction before the addEvent.

Reply:
Sorry, I did not check the oracle text. This card needs a RevealAction as it is stated in the oracle text. It should both a LookAction and a RevealAction.